src/couldBeSharedComponents/LayoutPanels/LayoutPanels.module.scss

```diff
@@ -18,12 +18,11 @@
   flex-grow: 1;
 }
 .panel {
-  transition: 1s ease-in-out;
```

Comment:
this is the new default animation @alanjleonard . When you add a bunch of styling defaults, its all just CSS, so you can tweak it however you see fit :)